### DISCUSSION TITLE
Previously working static query suddenly broke.  Here's what fixed it.

### DISCUSSION THREAD

**Discussion by Ukiman1014** [microsoft-search/pnp-modern-search#2699] [General]
A static query that had been working for months suddenly broke last week (or someone noticed that it broke).  After some trial and error, I got it working again by removing the quotes from "*".  I've no idea why it suddenly stopped working.  Just thought I'd post this here in case anyone else has the same issue, or maybe some info on why I had to make the change.  Did SharePoint search change recently?

![2023-01-17_121117](https://user-images.githubusercontent.com/22643106/212969431-ab362110-d226-4b4a-9e40-9e864be61175.jpg)

**Comment by wobba** [comment]
Thank you.. and from a semantic point of view `"*"` would be a phrase with the term `*` in it, not `*` (wildcard). Might be they worked the same earlier but I think it's ok they don't.

**Reply by Ukiman1014** [reply]
I agree with you.  It's just odd that the results suddenly changed last week.  Thankfully, it was an easy fix.  One user saw a correlation ID error but I couldn't get Microsoft to look it up for me. 🤷🏻‍♂️
